tutorials/tts/FastPitch_MultiSpeaker_Pretraining.ipynb

```diff
@@ -0,0 +1,867 @@
+    "    if isinstance(audio, torch.Tensor):\n",
+    "        audio = audio.to('cpu').numpy()\n",
+    "    return audio"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "96fcb433",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Reference Audio\n",
+    "reference_records = []\n",
+    "with open(train_manifest, \"r\") as f:\n",
+    "    for i, line in enumerate(f):\n",
+    "        reference_records.append(json.loads(line))\n",
+    "\n",
+    "speaker_to_index = defaultdict(list)\n",
+    "for i, d in enumerate(reference_records): speaker_to_index[d.get('speaker', None)].append(i)\n",
+    "        \n",
+    "# Validatation Audio\n",
+    "num_val = 3\n",
+    "val_records = []\n",
+    "with open(valid_manifest, \"r\") as f:\n",
+    "    for i, line in enumerate(f):\n",
+    "        val_records.append(json.loads(line))\n",
+    "        if len(val_records) >= num_val:\n",
+    "            break"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7ae8f4e3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for i, val_record in enumerate(val_records):\n",
+    "    reference_record = reference_records[speaker_to_index[val_record['speaker']][0]]\n",
+    "    reference_spec, reference_spec_lens = gt_spectrogram(reference_record['audio_filepath'], wave_model, spec_model)\n",
+    "    reference_spec = reference_spec.to(spec_model.device)\n",
+    "    spec_pred = gen_spectrogram(val_record['text'], \n",
+    "                                spec_model,\n",
+    "                                speaker=val_record['speaker'], \n",
+    "                                reference_spec=reference_spec, \n",
+    "                                reference_spec_lens=reference_spec_lens)\n",
+    "\n",
+    "    audio_gen = synth_audio(vocoder_model, spec_pred)\n",
+    "    \n",
+    "    audio_ref = ipd.Audio(reference_record['audio_filepath'], rate=sample_rate)\n",
+    "    audio_gt = ipd.Audio(val_record['audio_filepath'], rate=sample_rate)\n",
+    "    audio_gen = ipd.Audio(audio_gen, rate=sample_rate)\n",
+    "    \n",
+    "    print(\"------\")\n",
+    "    print(f\"Text: {val_record['text']}\")\n",
+    "    print('Reference Audio')\n",
+    "    ipd.display(audio_ref)\n",
+    "    print('Ground Truth Audio')\n",
+    "    ipd.display(audio_gt)\n",
+    "    print('Synthesized Audio')\n",
+    "    ipd.display(audio_gen)\n",
+    "    plt.imshow(spec_pred[0].to('cpu').numpy(), origin=\"lower\", aspect=\"auto\")\n",
+    "    plt.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "884e6906",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "str(YOUR_PRETRAINED_FASTPITCH_CHECKPOINT)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8d17f232",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "str(YOUR_FINETUNED_HIFIGAN_ON_MULTISPEAKER_CHECKPOINT)"
```

Comment:
Can remove the stray prints at the end

Reply:
I use this to let user know which checkpoint they'll get after running this tutorial.

Reply:
Ok, then how about we make it slightly more descriptive and put it in 1 cell.

```
print(f"FastPitch checkpoint: {YOUR_PRETRAINED_FASTPITCH_CHECKPOINT}")
print(f"HiFi-Gan checkpoint: {YOUR_FINETUNED_HIFIGAN_ON_MULTISPEAKER_CHECKPOINT}")
```